src/python/WMCore/WMSpec/Steps/Executors/CMSSW.py

```diff
@@ -365,7 +365,18 @@ def post(self, emulator=None):
 if [ $? -ne 0 ]; then echo "***\nCouldn't get scram runtime: $?\n*"; exit 73; fi
```

Comment:
There are more problems, line like this Couldn't get scram runtime: $?\n will return `Couldn't get scram runtime: 0`. 

`To check the exit code we can simply print the $? special variable in bash. This variable will print the exit code of the last run command.` In this case last run command is not eval....